webpack.config.js

```diff
@@ -32,7 +45,11 @@ const config = {
     libraryExport: 'default',
     clean: true
   },
-  externalsType: 'umd',
+  resolve: {
+    extensions: ['.js', '.vue'],
+    symlinks: false
+  },
+  externalsType: 'commonjs',
```

Comment:
Why is this not set based on `env.addon`?

Reply:
the externals can always be set to commonjs and it still works they do not need to be in umd format. Since I prefer a solution to avoind the usage of the umd format completely this was a good starting point to reduce all occurences of such